src/components/availability-label/availability-label.tsx

```diff
@@ -13,79 +14,90 @@ export interface AvailabilityLabelProps {
   handleSelectManifestation?: () => void | undefined;
   cursorPointer?: boolean;
 }
+export const AvailabilityLabel: React.FC<AvailabilityLabelProps> = withConfig(
+  ({
+    manifestText,
+    selected = false,
+    url,
+    faustIds,
+    handleSelectManifestation,
+    cursorPointer = false
+  }) => {
+    const config = useConfig();
+    const blacklistBranches = config("blacklistedAvailabilityBranchesConfig", {
+      transformer: "stringToArray"
+    });
 
-export const AvailabilityLabel: React.FC<AvailabilityLabelProps> = ({
-  manifestText,
-  selected = false,
-  url,
-  faustIds,
-  handleSelectManifestation,
-  cursorPointer = false
-}) => {
-  const t = useText();
-  const { data, isLoading, isError } = useGetAvailabilityV3({
-    recordid: faustIds
-  });
+    const t = useText();
+    const { data, isLoading, isError } = useGetAvailabilityV3({
+      recordid: faustIds,
+      ...(Array.isArray(blacklistBranches)
+        ? { exclude: blacklistBranches }
+        : {})
+    });
```

Comment:
With the latest changes to `useConfig()`, `blacklistBranches` should always return an array of strings so either there is no need for this `Array.isArray()` check or the check should be different - e.g. checking for the length if we only want to pass the `exclude` parameter if there are any blacklisted branches configured.